Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/Callout.qml

```diff
@@ -164,9 +198,9 @@ Item {
     /*! \internal */
     property real calloutMaxWidth: 210
     /*! \internal */
-    property real calloutMaxHeight: 100
+    property real calloutMaxHeight: 50
     /*! \internal */
-    property real calloutMinWidth: calloutMaxWidth
+    property real calloutMinWidth: 95
```

Comment:
No we do it later here: https://github.com/Esri/arcgis-runtime-toolkit-qt/blob/v.next/Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/Callout.qml#L272